mbuild/lattice.py

```diff
@@ -387,6 +432,7 @@ def _from_lattice_vectors(self):
 
         _from_lattice_vectors will calculate the angles alpha, beta, and
         gamma from the Lattice object attribute lattice_vectors.
+
         """
 
         degreeConvsersion = 180.0 / np.pi
```

Comment:
I just noticed my spelling mistake here.

@daico007 could you change these in this function?
```suggestion
        degreeConversion = 180.0 / np.pi
```